test/integration/amqpevents.test.js

```diff
@@ -0,0 +1,128 @@
+const amqp = require('amqplib');
+
+const sinon = require('sinon');
+const { expect } = require('chai');
+
+const {
+  AMQPEventsSender,
+  AMQPEventsReceiver
+} = require('../../');
+
+/**
+ * Helper method, returns promise, that fulfills
+ * when event is emitted on eventEmitter;
+ * @param {String} event
+ * @param {EventEmitter} eventEmitter
+ * @returns {Promise}
+ */
+function waitFor(event, eventEmitter) {
+  return new Promise((res) => eventEmitter.on(event, res));
+}
+
+describe('AMQPEventsSender AMQPEventsReceiver', () => {
+  let amqpConn;
+
+  beforeEach(async () => {
+    amqpConn = await amqp.connect('amqp://localhost');
```

Comment:
(not critical for me) - Yarik requested changes from my PR to move these in separate helpers, so tests don't know about connection, but knows about helper that handles the connection. So, IDK, for me it's ok :)